backend/benefit/messages/permissions.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+from rest_framework import permissions
+from users.utils import get_company_from_request
+
+
+class IsMessageAuthor(permissions.BasePermission):
```

Comment:
Maybe reconsider the name of the class. For applicants, we're not checking for authorship, but company

Reply:
Make sense, I changed the name to `HasMessagePermission`